src/__tests__/role.js

```diff
@@ -564,11 +565,11 @@ describe('configuration', () => {
       queryByRole('button', {name: 'Hello, Dave!'})
 
       expect(console.error).toHaveBeenCalledTimes(2)
-      expect(console.error.mock.calls[0][0]).toMatch(
-        'Error: Not implemented: window.computedStyle(elt, pseudoElt)',
+      expect(console.error.mock.calls[0][0].message).toMatch(
+        'Not implemented: window.computedStyle(elt, pseudoElt)',
```

Comment:
JSDOM now calls `console.error` with the actual `Error` instance instead of its stringified version